types/xsalsa20-encoding/index.d.ts

```diff
@@ -0,0 +1,130 @@
+// Type definitions for xsalsa20-encoding 1.1
+// Project: https://github.com/baz/foo (Does not have to be to GitHub, but prefer linking to a source code repository rather than to a project website.)
```

Comment:
this should link to the source repo or project web site. If there isn't one, github.com/DefinitelyTyped/DefinitelyTyped will do.

Reply:
done